### ISSUE TITLE
LDAP group_search_base_dns is being handled wrong

### ISSUE THREAD

**Issue by jblaine** [sous-chefs/grafana#279]
Cookbook 5.1.1 on RHEL 7 and Chef client 14.10 (doesn't matter in this case).

TL;DR : Pretty sure group_search_base_dns is supposed to be implemented as an Array, not String.

Longer:

Even though [search_base_dns](https://github.com/sous-chefs/grafana/blob/master/resources/config_ldap_servers.rb#L36) is an Array, the docs and [code](https://github.com/sous-chefs/grafana/blob/master/resources/config_ldap_servers.rb#L38) have  [group_search_base_dns](https://github.com/sous-chefs/grafana/blob/master/resources/config_ldap_servers.rb#L38) as a **String**.

Docs:
```
group_search_base_dns
String
POSIX, An array of the base DNs to search through for groups. Typically uses ou=groups
```

When implemented as a string per the docs, LDAP authentication throws a TOML error.
```
grafana_config_ldap_servers 'grafana' do
...
  search_base_dns ['dc=our,dc=org']
  search_filter '(&(objectCategory=Person)(sAMAccountName=%s))'
  group_search_base_dns 'dc=our,dc=org'
...
end
```
Error:
```t=2019-08-17T14:13:44-0400 lvl=eror msg="Error while trying to authenticate user" logger=context userId=0 orgId=0 uname= error="Failed to get LDAP config: Failed to load LDAP config file: Near line 26 (last key parsed 'servers.group_search_base_dns'): expected value but found \"dc\" instead"```

This is because the string is rendered in TOML without quotes / not as a string:

```group_search_base_dns = dc=our,dc=org```

Regardless, I suspect the bottom-line problem is that group_search_base_dns is supposed to be implemented and documented as type Array, not String.

**Comment by xorima** [comment]
Hey @jblaine 
Any chance you can sort out a PR for this then we can get a fix out quickly?

Thanks!

**Comment by jblaine** [comment]
@Xorima PR submitted.
